srfi-tools/generate.scm

```diff
@@ -5,6 +5,169 @@
 (define (faq-anchor-template description name)
   `(a (@ (href ,(concat "/srfi-faq.html#" name))) ,description))
 
+;; Format string for srfi-NNN.html.
+;; TODO: Convert to SXML.
+(define srfi-html-template "<!DOCTYPE html>
+<html lang=\"en\">
+  <head>
+<!--
+SPDX-FileCopyrightText: ~a ~a
+SPDX-License-Identifier: MIT
+-->
+    <meta charset=\"utf-8\">
+    <title>SRFI ~a: ~a</title>
+    <link href=\"/favicon.png\" rel=\"icon\" sizes=\"192x192\" type=\"image/png\">
+    <link rel=\"stylesheet\" href=\"https://srfi.schemers.org/srfi.css\" type=\"text/css\">
+    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">
+    <style>
+     small { font-size: 14px; vertical-align: 2px; }
+     body  { line-height: 24px; }
+     pre  { font-family: inherit; line-height: 20px; }
+    </style>
+  </head>
+  <body>
+    <h1><a href=\"https://srfi.schemers.org/\"><img class=\"srfi-logo\" src=\"https://srfi.schemers.org/srfi-logo.svg\" alt=\"SRFI surfboard logo\" /></a>~a: ~a</h1>
+
+<p>by ~a</p>
+
+<h2 id=\"status\">Status</h2>
+
+    <p>
+      This SRFI is currently in <em>~a</em> status.
+      Here is <a href=\"https://srfi.schemers.org/srfi-process.html\">an explanation</a> of each status that a SRFI can hold.
+      To provide input on this SRFI, please send email to <code><a href=\"mailto:srfi+minus+~a+at+srfi+dotschemers+dot+org\">srfi-~a@<span class=\"antispam\">nospam</span>srfi.schemers.org</a></code>.
+      To subscribe to the list, follow <a href=\"https://srfi.schemers.org/srfi-list-subscribe.html\">these instructions</a>.
+      You can access previous messages via the mailing list <a href=\"https://srfi-email.schemers.org/srfi-~a/\">archive</a>.
+    </p>
+
+    <ul>
+      <li>Received: ~a</li>
+~a
+      ~a
+    </ul>
+
+<h2 id=\"abstract\">Abstract</h2>
+
+~a
+
+~a
+
```

Comment:
Don't forget the other required sections.  The ones missing here are:

- Issues
- Specification
- Implementation
- Acknowledgements

Reply:
Everything after "Rationale" is extracted, so Specification, Implementation, and Acknowledgements are already taken care of (along with other arbitrary sections the author may have added).

I will update this PR to make sure "Issues" is extracted as well.